src/platform-apps/channels/util/api.test.ts

```diff
@@ -1,12 +0,0 @@
-import { searchMentionableUsersForChannel } from './api';
-
-describe('userMentionSearch', () => {
-  it('maps to user mentions format', async function () {
-    const user1 = { id: 'd-1', name: 'dale', profileImage: 'http://example.com' };
-    const apiSearch = jest.fn().mockResolvedValue([user1]);
-
-    const searchResults = await searchMentionableUsersForChannel(null, null, apiSearch);
-
-    expect(searchResults).toEqual([{ id: 'd-1', display: 'dale', profileImage: 'http://example.com' }]);
-  });
-});
```

Comment:
wasn't sure about how to re-add the test here. Since i moved the entire logic into a single function in the class 🤔

Reply:
I'm going to go with, "it doesn't matter because we're deprecating this anyway"

Reply:
got it